.semaphore/semaphore.yml

```diff
@@ -18,7 +22,28 @@ blocks:
           - cache store
           - cp /Users/semaphore/BeeSwift/BeeSwift/Config.swift.sample /Users/semaphore/BeeSwift/BeeSwift/Config.swift
       jobs:
-        - name: bundle exec fastlane test
+        - name: Test
           commands:
             - bundle exec xcversion select 11.3.1
             - bundle exec fastlane test
+  - name: Fastlane Build
+    task:
+      prologue:
+        commands:
+          - checkout
+          - cache restore
+          - bundle install --path vendor/bundle
+          - cache store
+          - cp /Users/semaphore/BeeSwift/BeeSwift/Config.swift.sample /Users/semaphore/BeeSwift/BeeSwift/Config.swift
+      jobs:
+        - name: Build
+          commands:
+            - bundle exec xcversion select 11.3.1
+            - bundle exec fastlane test
```

Comment:
besides the env_var section being placed elsewhere, the content of this job/task seems to be the same as the other. What does 'fastlane build' provide that 'fastlane test' does not?

Reply:
that was a typo :) fixed now